### ISSUE TITLE
Not a "Feature Request" but tech question about compression algorithms

### ISSUE THREAD

**Issue by ljalan1776** [RandomEngy/VidCoder#1084]
### Feature details

Hi. I am downloading a bunch of videos that are 2gb for 10 minutes. If I upload it to youtube and let youtube's algorithm compress it the downloaded file comes out between 80 and 150 megs depending on if I go with 720P or 1080P. I usually go with 720. 

My vids aren't youtube friendly though and even though I upload them to youtube on private my accounts been flagged (it's OK, it's a throw away account). 

I was wondering if someone can post some settings for VidCoder that will do the same type of compression as Youtube? I am not familiar with how to look at the technical details or input them into VidCoder (I know how to do a custom compression, I just don't know what values to input). 

Thank you for your advice. 

In a nutshell: my goal is = smallest file size without losing noticable quality. 

For further tech specs all of my videos are mp4s but they're obviously hardly compressed. 
Thank you for taking the time to post any recommendations.

**Comment by RandomEngy** [comment]
There's a lot of considerations and complexity you can get into, but you'll get good results if you stick to a few considerations:

1) Use quality targeting. Experiment with different quality levels and pick one you find acceptable that's got the lowest size. I'm pretty sure that's how YouTube does it because if you download a video with a static picture, it's really small; basically the size of the audio.
2) Use x265 if your playback devices support it. You get much smaller sizes that way.
3) Use the presets; don't get into the weeds tweaking super-advanced encoding settings. Basically everyone who does this ends up with something worse than one of the presets. The presets trade off encoding time for encoding efficiency but note that you get sharply diminishing returns as you bring the slider to the "slow" side.
4) Check your filters and avoid resizing the picture if you can. Resampling pixels can degrade quality. An exception would be if the source video is a far higher resolution than your playback device, in which case you can resize down for a significant size saving.
